### ISSUE TITLE
Led animation function does not exist in ARDroneDLL

### ISSUE THREAD

**Issue by miguelbarros** [shtejv/ARDrone-Control-.NET#4]
Although the led animation is implemented in led_animation.h and through ardrone_at_set_led_animation, it is not possible to run animations from c# code. This feature could be usefull for initial dev tests - or even for allowing the drone to communicate with the user.

**Comment by theone1984** [comment]
This is a nice feature that we could implement in one of the next versions of the library.

**Comment by miguelbarros** [comment]
I tried implementing it locally by creating a new export on the c++ and adding the dllimport on the ARDroneControl side. It sort of worked - I was able to animate the leds. However this only works for the first animation. If I make a second call to change the animation again it won't work unless I reconnect to the drone (by restarting the application). I'm probably missing some kind of reset command - unfortunately the parrot documentation for the leds is not very helpfull...
